Makefile

```diff
@@ -839,6 +839,7 @@ LIB_OBJS += base85.o
 LIB_OBJS += bisect.o
 LIB_OBJS += blame.o
 LIB_OBJS += blob.o
+LIB_OBJS += bloom.o
```

Comment:
Commit note: this commit message includes a lot of RFC comments and questions. The biggest one is "should we include documentation for the file format?" I believe we _will_ need that documentation, but it should be a commit before this one. That can be converted to a TODO in the cover letter.

Reply:
I also believe there was a plan to split this into three commits:

1. Implement and unit-test the murmur3 hash algorithm.
2. Implement and unit-test the Bloom key computation (given some `data` string).
3. Call the Bloom filter computation from the commit-graph code on write.